docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -0,0 +1,264 @@
+---
+approvers:
+- mtaufen
+- dawnchen
+title: Reconfigure a Node's Kubelet in a Live Cluster
+---
+
+{% capture overview %}
+As of Kubernetes 1.8, the new [Dynamic Kubelet Configuration](https://github.com/kubernetes/features/issues/281) feature is available in alpha. This allows you to change the configuration of Kubelets in a live Kubernetes cluster via first-class Kubernetes concepts. Specifically, this feature allows you to configure individual Nodes' Kubelets via ConfigMaps.
+
+**Warning:** The organization of the Kubelet's configuration type is still considered alpha, and it is likely to be restructured with breaking changes before graduating from alpha.
+
```

Comment:
Add this line:
{% include feature-state-alpha.md %}

Then you can probably delete the first of your two Warnings.

Reply:
Just include it right at the bottom?

Reply:
The second warning is worth leaving IMO, we don't want users to assume this implies all fields are safe to change without care.

Reply:
Put {% include feature-state-alpha.md %} right after {% capture overview %}. Yes, I think we should keep the second warning.